martor/static/martor/js/martor.js

```diff
@@ -53,7 +53,7 @@
             var editor       = ace.edit(editorId);
             var editorConfig = JSON.parse(textareaId.data('enable-configs').replace(/'/g, '"'));
 
-            if (darkMode) {
+            if (darkMode && !field_name.match(/^reply-/)) {
```

Comment:
Not convinced this is sufficiently generic. I think you should check for `inverted` being on a known class or not.

Reply:
`addClass` should work here - https://stackoverflow.com/a/7403519

`inverted` isn't used for light mode, so there is no need for `toggleClass`

Reply:
`inverted` was used in light mode in the emoji picker, but that has since been deleted, so `addClass` is now sufficient.